ipetcare.spa/src/features/auth/containers/loginForm.tsx

```diff
@@ -23,17 +24,12 @@ export function LoginForm() {
     e.preventDefault()
```

Comment:
Do czego tak właściwie nam tutaj e.preventDefault? Pytam bo nie wiem :3

Reply:
dużo rzeczy z automatu się robi i to im zapobiega, żeby się nie robiły (chyba jakieś walidacje itp, nie pamiętam)